tasks/56_search-and-replace/search-and-replace.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * myReplace - Perform a search and replace on the sentence
+ * using the arguments provided and return the new sentence.
+ *
+ * @example
+ * //return 'Let us go to the mall'
+ * myReplace("Let us go to the store", "store", "mall");
+ * @param {String} str - source sentence.
+ * @param {String} before - sentence to perform the search and replace on.
+ * @param {String} after - sentence to replace.
+ * @return {String} new sentence after replacing.
+ */
+export function myReplace(str, before, after) {
```

Comment:
You described variables in JSDoc pretty good but why did you name them str, before and after?
`export function myReplace (source, searchValue, replacer)`

Reply:
@KalachevDev  i use names from examples in https://docs.google.com/document/d/19lKS_3-isMu7AWxlZF4rVF1EGnJIsg8YtYzS2D99sSs/